src/node-red/hardware/revolutionary-pi.md

```diff
@@ -0,0 +1,155 @@
+```
+
+## Getting Started
+
+This guide provides step-by-step instructions for setting up the FlowFuse Device Agent on a Revolutionary Pi. It begins with installing Node.js, a prerequisite for running the agent, followed by the installation and configuration of the FlowFuse Device Agent.
+
+### Installing Node.js
+
+Since Node.js is not preinstalled on the Revolutionary Pi, follow these steps to install it:
+
+#### Step 1: Update System Packages
+
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+
+#### Step 2: Install Node.js
+
+You can install Node.js from the default APT repositories:
+
+```bash
+sudo apt install -y nodejs
+```
+
+#### Step 3: Verify Installation
+
+```bash
+node -v
+```
+
+After running the verification command, the installed Node.js version should be displayed in the terminal.
+
+> **Note:** Ensure that the installed version is **Node.js 18 or later**, as this is the recommended version for optimal compatibility with the device agent.
+
+### Installing the FlowFuse Device Agent
+
+#### Step 1: Install Required Package (curl)
+
+If `curl` is not already installed on your system, you will need to install it first:
+
+```bash
+sudo apt install -y curl
+```
+
+#### Step 2: Execute the Installation Script
+
+To install the FlowFuse Device Agent and set it up as a service, download and execute the installation script from GitHub:
+
+```bash
+bash <(curl -sL https://raw.githubusercontent.com/FlowFuse/device-agent/main/service/raspbian-install-device-agent.sh)
+```
+
+This script installs the agent, and configures the Revolutionary Pi to run the FlowFuse agent on boot and restart it if it crashes.
+
+### Registering the Device Agent to FlowFuse and Running It
+
+After installing the FlowFuse Device Agent, the next step is to link the Revolutionary Pi to your [FlowFuse team](/docs/user/team/). This allows the device to be managed remotely and receive Node-RED deployments.
+
+#### Step 1: Add a New Device to Your FlowFuse Team
+
+Follow the instructions in the official documentation to add your device to your FlowFuse team: [Add a Remote Instance](/docs/device-agent/register/#add-remote-instance).
+
+Once you register the device, you will receive the configuration details required to connect device to your team. Copy the provided command.
+
+![Image showing command device configuration dialog and the command placeholder, where you will find the command to link the device to your FlowFuse team.](./images/how-to-setup-node-red-on-raspberry-pi-device-configuration-dialog.png "Image showing command device configuration dialog and the command placeholder, where you will find the command to link the device to your FlowFuse team."){data-zoomable}
+
+#### Step 2: Configuring the Device
+
+Execute the command you copied to complete the device configuration.
+
+#### Step 3: Restart the Device Agent
+
+Once the device is configured, restart it. The FlowFuse Device Agent will start automatically upon reboot.
```

Comment:
<obtain connection command, link to external docs page>


```bash
flowfuse-device-agent --quick-connect <insert-your-three-word-token>
sudo systemctl start flowfuse-device-agent
```

Verify in FlowFuse that your device is connected.